.github/workflows/run_tests.yaml

```diff
@@ -127,13 +127,12 @@ jobs:
           if-no-files-found: ignore
           retention-days: 3
 
-      - name: Archive runned cypress tests videos result
+      - name: Archive runned cypress tests result
         uses: actions/upload-artifact@v3
-        if: ${{ failure() }}
+        if: ${{ always() }}
```

Comment:
Is it wrong result from rebase procedure? Try to compare code with main branch, please.

https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/912c24c7b38c69f290d63f7e7e31ce40c915b11a/.github/workflows/run_tests.yaml#L120-L136

Reply:
This is removed.